integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala

```diff
@@ -171,19 +170,25 @@ class CarbonSession(@transient val sc: SparkContext,
    */
   private def trySearchMode(qe: QueryExecution, sse: SQLStart): DataFrame = {
     val analyzed = qe.analyzed
+    val LOG: LogService = LogServiceFactory.getLogService(classOf[CarbonSession].getName)
     analyzed match {
       case _@Project(columns, _@Filter(expr, s: SubqueryAlias))
         if s.child.isInstanceOf[LogicalRelation] &&
            s.child.asInstanceOf[LogicalRelation].relation
              .isInstanceOf[CarbonDatasourceHadoopRelation] =>
+        LOG.info(String.format("Search service started and supports: %s", sse.sqlText))
```

Comment:
It is in scala here, why still use String.format?

Reply:
ok, thanks, changed